stable/mongodb-replicaset/init/files/add-name-label.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function create-patch-file () {
```

Comment:
This is very very hacky. Can we instead just use kubectl in a pod or something?

Reply:
I like that idea. Would be much cleaner.

@mariomarin Here's an example how this can be done: https://github.com/coreos/prometheus-operator/blob/d9073040bdf15c02ef0e2ef7216c80dde6dd71fd/helm/prometheus-operator/templates/get-tprs-job.yaml#L21-L30